mos_tests/rabbitmq_oslo/conftest.py

```diff
@@ -38,3 +44,43 @@ def fixt_kill_rpc_server_client(env):
         with node.ssh() as remote:
             cmd = 'pkill -f oslo_msg_check_'
             remote.execute(cmd)
+
+
+@pytest.fixture
+def controller(env):
+    return random.choice(env.get_nodes_by_role('controller'))
+
+
+@pytest.yield_fixture
+def patch_iptables(controller, request):
+    """Apply IPTables rules"""
+
+    def add_ports(tmpl):
+        ports = [4369, 5672, 5673, 25672]
+        return ' '.join(map(lambda portnum: tmpl.format(portnum=portnum),
+                            ports))
+
+    if request.param == 'drop':
```

Comment:
Let's use 'DROP' and 'REJECT' as request.param and templates will looks like
'iptables -I INPUT -p tcp -m tcp --dport {portnum} -j {action} ;',
where action will pass directly from request.param